blocks/hero-block/features/hero/default.jsx

```diff
@@ -0,0 +1,166 @@
+
+	return (
+		<div className={classes}>
+			<picture>
+				<source srcSet={imageURLDesktop} media={`(${desktopMediaBreakpoint})`} />
+				<img
+					className={
+						layout === "stacked"
+							? `${BLOCK_CLASS_NAME}__image--stacked`
+							: `${BLOCK_CLASS_NAME}__image`
+					}
+					src={imageURLMobile}
+					alt={imageAltText}
+				/>
+			</picture>
+
+			<Stack
+				className={`${BLOCK_CLASS_NAME}__text--${layout} ${
+					alignment === "left"
+						? `${BLOCK_CLASS_NAME}__text--left`
+						: `${BLOCK_CLASS_NAME}__text--center`
+				}`}
+			>
+				<HeadingWrapper>
+					{headline ? <Heading>{headline}</Heading> : null}
+					{subHeadline ? (
+						<HeadingSection>
+							<Heading className={`${BLOCK_CLASS_NAME}__subheadline`}>{subHeadline}</Heading>
+						</HeadingSection>
+					) : null}
+					{description ? <Paragraph>{description}</Paragraph> : null}
+
+					{(link1Action && link1Text) || (link2Action && link2Text) ? (
+						<Stack
+							className={`${BLOCK_CLASS_NAME}__buttonrow`}
+							direction="horizontal"
+							justification={alignment === "center" ? "center" : "start"}
+						>
+							{link1Action && link1Text ? (
+								<Button href={link1Action} variant={link1Type} size="large">
+									{link1Text}
+								</Button>
+							) : null}
+							{link2Action && link2Text ? (
+								<Button href={link2Action} variant={link2Type} size="large">
+									{link2Text}
+								</Button>
+							) : null}
+						</Stack>
+					) : null}
+				</HeadingWrapper>
+			</Stack>
+		</div>
+	);
+}
+
+Hero.label = "Hero – Arc Block";
+
+Hero.icon = "picture-double-shapes";
+
+Hero.propTypes = {
+	customFields: PropTypes.shape({
+		layout: PropTypes.oneOf(["overlay", "stacked"]).tag({
+			defaultValue: "overlay",
+			label:
+				"Layout configuration. Overlay is text on top of the image and stacked is the text under the image.",
+		}),
+		alignment: PropTypes.oneOf(["center", "left"]).tag({
+			defaultValue: "center",
+			label: "Content Alignment over the image. Applies to overlay layout only. ",
+		}),
+		variant: PropTypes.oneOf(["dark", "light"]).tag({
+			defaultValue: "dark",
+			label: "Content Style",
+		}),
+		imageURLDesktop: PropTypes.string.tag({
+			label: "Image URL for Desktop",
+			searchable: "image",
+		}),
+		desktopMediaBreakpoint: PropTypes.oneOf([
+			"min-width: 800px",
+			"min-width: 700px",
+			"min-width: 600px",
+			"min-width: 500px",
+		]).tag({
+			defaultValue: "min-width: 800px",
+			label:
+				"Image Breakpoint: Enter the media query that will determine when the desktop picture is shown over the mobile version.",
+		}),
+		imageURLMobile: PropTypes.string.tag({
+			label: "Image URL for Mobile",
+			searchable: "image",
+		}),
+		imageAltText: PropTypes.string.tag({
+			defaultValue: "",
+			label: "Alt text for both the mobile and desktop image",
+		}),
+		headline: PropTypes.string,
+		subHeadline: PropTypes.string,
+		description: PropTypes.string,
```

Comment:
These should have client-facing labels like from ticket: 

    * Headline Text <required field>
    * Sub-Headline Text <optional field>
    * Description Text <optional field>
    * Button #1 Text <required field>
    * Button #1 Link Url <required field>
    * Button #2 Text <optional field>
    * Button #2 Link Url <optional field>

Reply:
<img width="244" alt="Image URL for Mobile" src="https://user-images.githubusercontent.com/5950956/162067265-6a24d909-55fa-49ba-b0fb-040891232f34.png">

Reply:
@visionbegin I would expect subHeadline label to be "Sub-Headline Text" per the ticket. don't need a description. I just noticed the lowercase and camelcase naming

Reply:
How's this?

![image](https://user-images.githubusercontent.com/833461/162235102-43879a99-3f80-4cc9-b11d-d20517395316.png)

Reply:
thanks !